signac/contrib/project.py

```diff
@@ -2384,7 +2382,7 @@ def get_job(cls, root=None):
 
         # Find the last match instance of a job id
         results = list(re.finditer(JOB_ID_REGEX, root))
-        if len(results) == 0:
+        if not results:
```

Comment:
While I also often use the boolean evaluation over the explicit length, I don't agree that this is necessarily more obvious to a (novice) reader.

Reply:
```suggestion
        if len(results) == 0:
```